backend/eveonline/routers/characters.py

```diff
@@ -205,3 +222,23 @@ def wrapped(request, token):
         return redirect(request.session["redirect_url"])
 
     return wrapped(request)  # pylint: disable=no-value-for-parameter
+
+
+@router.post(
+    "/{int:character_id}/tags",
+    summary="Get character tags by ID",
+    auth=AuthBearer(),
+    response=None,
+)
+def post_character_tags(request, character_id: int, tag_req: TagRequest):
+    character = EveCharacter.objects.get(character_id=character_id)
+    if (
+        request.user.has_perm("eveonline.change_evecharacter")
+        or Token.objects.filter(
+            user=request.user, character_id=character_id
+        ).exists()
+    ):
+        for tag in tag_req.tags:
+            EveCharacterTag.objects.create(character=character, tag=tag)
+        return 204
+    return 403, "You do not have permission to view this character."
```

Comment:
I don't think this will work, I ran into a similar issue, see how `get_character_by_id` handles the exceptions / error cases and how the decorator defines responses